recipes/meggie/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "meggie" %}
+{% set version = "1.3.0" %}
+{% set description = "User-friendly graphical user interface to do M/EEG analysis" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e052e8e50884381c8de06e915b18ab22a27d42dc806f2da2fb937d2f5dfa0c0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - mne >=0.23.0
+    - matplotlib
+    - scikit-learn
+    - python-json-logger
+    - pyqt >=5
+
+test:
+  imports:
+    - meggie
+  commands:
+    - pip check
+  requires:
+    - pip
+
+app:
+  entry: meggie
+  summary: {{ description }}
```

Comment:
Please put the description here in plain and avoid unnecessary Jinja2 variables.